clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java

```diff
@@ -107,7 +107,7 @@ public void testResolveDnsLookup() throws UnknownHostException {
 
     @Test
     public void testResolveDnsLookupAllIps() throws UnknownHostException {
-        assertEquals(2, ClientUtils.resolve("kafka.apache.org", ClientDnsLookup.USE_ALL_DNS_IPS).size());
+        assertTrue(ClientUtils.resolve("kafka.apache.org", ClientDnsLookup.USE_ALL_DNS_IPS).size() > 1);
```

Comment:
does this patch affect this unit test?

Reply:
No, these DNS tests are failing for current branch as well, cherry-picked the upstream fix apache#9294